spec/lago/api/connection_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Lago::Api::Connection do
+  subject(:connection) do
+    described_class.new(
+      'fake-api-key',
+      URI('https://testapi.example.org/')
+    )
+  end
+
+  context "when an unsuccessful request is made" do
```

Comment:
We prefer using single-quoted strings when there is no string interpolation. Can you please replace it in this file?

Reply:
👍